python/src/NAIS_doc.i.in

```diff
@@ -0,0 +1,132 @@
+%feature("docstring") OT::NAIS
+
+"Non Adaptive Importance Sampling (NAIS) algorithm.
+
+Parameters
+----------
+event : :class:`~openturns.RandomVector`
+    Event we are computing the probability of failure.
+
+rhoQuantile : float  :math:`0<\rho<1`
+    Intermediate quantile level.
+
+Notes
+-----
+The following explanations are given for a failure event defined as :math:`g(\mathbf{X})<T` with :math:`\mathbf{X}` 
+a random vector following a joint PDF :math:`f_\mathbf{X}`, :math:`T` a threshold and :math:`g` a limit state function,
+without loss of generality.
+
+The Importance Sampling (IS) probability estimate by Importance Sampling :math:`\widehat{P}^\text{IS}` is given by:
+
+.. math::
+
+    \widehat{P}^\text{IS}=\frac{1}{N} \sum_{i=1}^{N} {\mathbf{1}}_{g(\mathbf{x}_i)<T} \frac{h_0(\mathbf{x}_i)}{h(\mathbf{x}_i)}.
+
+with :math:`h_0 = f_\mathbf{X}` the PDF of :math:`\mathbf{X}`, :math:`h` the auxiliary PDF of Importance Sampling, 
+:math:`N`the number of independent samples generated with :math:`h` and :math:`{\mathbf{1}}_{g(\mathbf{x}_i)<T}` the 
+indicator function of failure domain. 
+
+The optimal density minimizing the variance of the estimator :math:`h_{opt}` is  defined as:
+
+.. math::
+
+    h_{opt}=\frac{{\mathbf{1}}_{g(x)<T}h_0}{P}.
+
+with :math:`P` the failure probability which is inaccessible in practice since this probability is the quantity of interest
+ and unknown. 
+
+The objective of Non parametric Adaptive Importance Sampling (NAIS) technique is to approximate the IS optimal auxiliary density
```

Comment:
best to move the bibliography link there: "... (NAIS) [morio2015]_ technique ... "
instead of the line "source: [morio2015]" at the end of the paragraph